logistics/preparation.md

```diff
@@ -0,0 +1,109 @@
+# Preparation
+
+1. Identify the Preparation Owner from the Partner/Customer:
+    - Determine who the responsible party is on the customer/partner side.
+    - Ensure they possess:
+      - Proper Azure Access.
+      - Adequate knowledge about Azure.
+      - Authorization to configure subscriptions.
+    
+1. Define/Create the Azure Subscriptions for the Hackathon:
+**Warning**: Costs will incur. [Monitor/manage expenses](https://learn.microsoft.com/en-us/azure/cost-management-billing/cost-management-billing-overview).
+    - Given the increased access for participants, ensure you use [sandbox subscriptions](https://learn.microsoft.com/en-us/azure/architecture/guide/azure-sandbox/azure-sandbox).  
+    **Note**: No new tenant is required. Create a new subscription within the existing tenant.  
+    - Identify hackathon participants. Determine:
+        - Existing subscriptions
+        - Visual Studio subscriptions
+        - Existing dev/test (sandbox) subscriptions
+    - Collaborate with the preparation owner to determine the number of subscriptions required. Considerations:
```

Comment:
Maybe we need to come up with a different term to preparation owner? Maybe hackathon environment organizer (something better than that though :D )